data/kinopub

```diff
@@ -0,0 +1,8 @@
+kino.pub
+kinopub.online
+gfw.ovh
+vjs.zencdn.net
+m.pushbr.com
+mos-gorsud.co
+zamerka.com
+regexp:(\w+)-static-[0-9]+\.cdntogo\.net$
```

Comment:
if the cdn domains can be enumerated, it is recommended not to use regexp.
Either wise, pls provide more info so I can confirm whether these domains are belong to kinopub